src/pychart/runner.py

```diff
@@ -42,7 +42,11 @@ def run_prompt():
                 print(result)
 
     except KeyboardInterrupt:
-        print("Keyboard Interrupt")
+        print("\nKeyboard Interrupted")
+        exit()
+        
+    except EOFError:
+        print()
```

Comment:
Why are we swallowing errors? Would be useful to know why the program exited for no reason